core/dbt/contracts/graph/manifest.py

```diff
@@ -960,6 +1008,13 @@ def metric_lookup(self) -> MetricLookup:
             self._metric_lookup = MetricLookup(self)
         return self._metric_lookup
 
+    @property
+    def semantic_model_by_measure_lookup(self) -> SemanticModelByMeasureLookup:
+        """Gets (and creates if necessary) the lookup utility for getting SemanticModels by measures"""
+        if self._semantic_model_by_measure_lookup is None:
```

Comment:
The checked-in code follows the paradigm of all the other `*_lookup` properties. I'm also biased because if we're specifically checking for `None`, then we should specifically check against `None`. Python can have many weird `false-y` states. It'd be weird if any cropped up here, but we are specifically checking against `None` intentionally